ui-playground/lib/background.js

```diff
@@ -0,0 +1,114 @@
+/* globals chrome:false */
+
+'use strict';
+
+chrome.browserAction.setBadgeText({text: '🚲ᕕ( ᐛ)ᕗ'});
+// chrome.browserAction.setBadgeBackgroundColor({color: '#FFF'});
+
+var chillOut = function (aTab) {
+  console.log("Chilling…");
+  // chrome.tabs.query({'url': '*://chilloutandwatchsomecatgifs.com/*'}, (tabs) => {
+  chrome.tabs.query({'url': 'http://chilloutandwatchsomecatgifs.com/'}, (tabs) => {
+    if (tabs.length) {
+      var tab = tabs.reduce((active, item) => {
+        if (aTab && item.id === aTab.id) {
+          active.push(item);
+        }
+        return active;
+      }, []);
+      if (tab.length === 0) {
+        chrome.tabs.update(tabs[0].id, {'active': true});
+        // chrome.pageAction.show(tabs[0].id);
+      } else {
+        // chrome.pageAction.show(tab[0].id);
+      }
+    } else {
+      var url = 'http://chilloutandwatchsomecatgifs.com/';
+      // var url = chrome.extension.getURL('catgifs.html') + '#blah:';
+      chrome.tabs.create({'url': url, 'active': true}, (tab) => {
+        console.log(tab);
+        // chrome.pageAction.show(tab.id);
+      });
+    }
+  });
+};
+
+var count = 0;
+
+chrome.notifications.onClicked.addListener(function (notificationId) {
+  // This doesn't work yet.
+  console.log("Notification Clicked!!!", notificationId);
+  chrome.tabs.query({active: true}, function (aTab) {
+    chillOut(aTab);
+  });
+});
+
+var audio = new Audio();
```

Comment:
I don't think any of this audio stuff works, because we don't support bookmarks.onCreated. I think examples should only contain things that work in their target version of Firefox.